doc/modules/calibration.rst

```diff
@@ -74,10 +74,14 @@ by showing the number of samples in each predicted probability bin.
 
 .. currentmodule:: sklearn.linear_model
 
-:class:`LogisticRegression` returns well calibrated predictions by default as it has a
+:class:`LogisticRegression` is more likely to return well calibrated predictions by itself as it has a
```

Comment:
Maybe:

```suggestion
Out of all the studied estimators, :class:`LogisticRegression` is the most likely to return well calibrated predictions by itself because:
- its training loss function is a `strictly proper scoring rule <https://en.wikipedia.org/wiki/Scoring_rule>`_,
- the regularization parameter `C` is tuned on the same loss via inner cross-validation,
- it uses canonical link function for its loss, i.e. the logit-link for the :ref:`log_loss`.
```

Note: the last item should be manually merge with the following line but I cannot do it via the GitHub suggestion UI.

Reply:
I can work on that in a follow-up PR.